examples/tabs-sync/src/tabs-sync.rs

```diff
@@ -42,6 +42,12 @@ fn main() -> Result<()> {
     cli_support::init_logging();
     let opts = Opts::from_args();
 
+    let (_, token_info) = match get_account_and_token(get_default_fxa_config(), &opts.creds_file) {
```

Comment:
I wonder if just using `?` here instead of explicitly handing the `Err` case works? Probably fine for an example. (Not that the changes here are wrong, just longer than might be necessary)